client/blocks/reader-post-card/style.scss

```diff
@@ -400,6 +406,13 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	}
 }
 
+// 3 line excerpt for thumbnail cards
+.reader-post-card.card.has-thumbnail:not(.is-gallery) {
```

Comment:
that (sadly) makes sense :(

maybe we should add a new type that gets applied in `client/state/reader/posts/normalization-rules.js`.

food for thought for the future!